qa/workunits/suites/ffsb.sh

```diff
@@ -4,10 +4,18 @@ set -e
 
 mydir=`dirname $0`
 
-wget http://download.ceph.com/qa/ffsb.tar.bz2
-tar jxvf ffsb.tar.bz2
-cd ffsb-6.0-rc2
-patch -p1 < $mydir/ffsb.patch
+# try it again if the clone is slow and the second time
+trap -- 'retry' EXIT
+retry() {
+    rm -rf ffsb
+    # double the timeout value
+    timeout 3600 git clone git://git.ceph.com/ffsb.git --depth 1
+}
+rm -rf ffsb
+timeout 1800 git clone git://git.ceph.com/ffsb.git --depth 1
```

Comment:
````
2021-04-30T19:42:38.166 INFO:tasks.workunit.client.0.smithi089.stdout:make[1]: Leaving directory '/home/ubuntu/cephtest/mnt.0/client.0/tmp/ffsb'
2021-04-30T19:42:38.200 INFO:tasks.workunit.client.0.smithi089.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/suites/ffsb.sh: line 27: ../ffsb-*/ffsb: No such file or directory
2021-04-30T19:42:38.201 DEBUG:teuthology.orchestra.run:got remote process result: 127
2021-04-30T19:42:38.201 INFO:tasks.workunit:Stopping ['suites/ffsb.sh'] on client.0...
````
From: /ceph/teuthology-archive/pdonnell-2021-04-30_06:53:29-fs-wip-pdonnell-testing-20210430.014813-distro-basic-smithi/6085949/teuthology.log

(and many others)

@lxbsz please run the ffsb workunit in tuethology to verify once you've fixed.

Reply:
Sure.

Reply:
For the lasted `ffsb` git repo we are switching to use, there has one commit will block our test. I have raised on PR https://github.com/ceph/ffsb/pull/2 to revert it.

For our test we set the `reuse=1` flag in `random-write.32.ffsb`, so it will fail directly here.

Reply:
Got it, you should be able to merge that PR now that you're in the CephFS team (I just invited you!).

Reply:
Yeah, done. thanks.